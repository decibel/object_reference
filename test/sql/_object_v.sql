\set ECHO none

\i test/load.sql

SELECT plan(
  0

  + 1 -- equality
);

-- TODO: load some damn data first
SELECT bag_eq(
  $$SELECT * FROM _object_reference._object_v__for_update$$
  , $$SELECT * FROM _object_reference._object_v$$
  , '_object_v__for_update matches _object_v'
);

\i test/pgxntool/finish.sql

-- vi: expandtab sw=2 ts=2
