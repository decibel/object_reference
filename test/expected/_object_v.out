\set ECHO none
1..1
ok 1 - _object_v__for_update matches _object_v
# TRANSACTION INTENTIONALLY LEFT OPEN!
